### PR TITLE
Fix: original-require package issues

### PR DESCRIPTION
**Pull request by xhulz**
## PR description

The `original-require` package was behaving unexpectedly when invoked. For this, an adjustment was made in `webpack` placing an alias ​​to trigger a script

Thank you @cds-amal and @davidmurdoch for your help on that :)

## Documentation

- [ ] I thought about documentation and added the `doc-change-required` label to this PR if documentation updates are required.

### CODE REVIEW

**Review by kevinbluer — APPROVED** [review]
Nice one @xhulz @cds-amal @davidmurdoch 👍

File: webpack.common.js [hunk 1]
```diff
@@ -27,6 +27,9 @@ module.exports = {
   },
   resolve: {
     // .json is added to prevent import error from /node_modules/got/index.js
+    alias: {
+      'original-require': require.resolve('./polyfills/original-require'),
+    },
```
Comment (cds-amal) [inline webpack.common.js:32]: This comment separates a comment from its target.

**Review by cds-amal — COMMENTED** [review]
@xhulz a small nit

File: webpack.common.js (hunk 1 above)
Comment (xhulz) [reply webpack.common.js:32]: Thank you @cds-amal! I'll fix that :)

**Pull request merged**